src/containers/RapidAppointment/RapidAppointment.tsx

```diff
@@ -343,9 +343,22 @@ export function RapidAppointment() {
       tagsCommaSeparatedString.push(vaccineTypeString);
     }
 
+    const tzOffset =
+      getTimezoneOffset(Intl.DateTimeFormat().resolvedOptions().timeZone) /
+      1000 /
+      60 /
+      60;
+
+    // This is a temporary hack while the backend figures out how to manage timezones properly
+    // at which point we will localize properly
+    const dateToSend = `${format(
```

Comment:
For rapid appointments while in EST, I am now passing: date: "2021-06-15T04:00:00+00:00"